torchtune/config/_parse.py

```diff
@@ -96,6 +95,6 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
         yaml_args, cli_args = parser.parse_known_args()
         conf = _merge_yaml_and_cli_args(yaml_args, cli_args)
 
-        sys.exit(recipe_main(conf))
+        recipe_main(conf)
```

Comment:
This is a change to all of Tune. Why is this necessary?

Reply:
Ahh was debugging - this can go back in.